cookiecutter.json

```diff
@@ -8,5 +8,7 @@
     "db_type": ["sqlite3", "postgres", "mysql"],
     "db_name": "Name of the database (ignore if you have selected sqlite3 as db_type)",
     "db_username": "Username for the database. (ignore if you have selected sqlite3 as db_type)",
-    "db_password": "passsword for the database. (ignore if you have selected sqlite3 as db_type)"
+    "db_password": "passsword for the database. (ignore if you have selected sqlite3 as db_type)",
+    "use_heroku": "n",
```

Comment:
add two options here for both use_heroku and use elasticbeanstalk
You can write like:
"use_heroku": ["yes","no"]